README.md

```diff
@@ -106,18 +106,18 @@ helpers.getShopperToken(config, { type: "guest" }).then(async (token) => {
 When using an IDE such as VSCode, the autocomplete feature lets you view the available method and class definitions, including parameters.
 ​
 
-![Autocomplete](https://github.com/SalesforceCommerceCloud/commerce-sdk/raw/master/images/Autocomplete.jpg?raw=true 'Autocomplete')
+![Autocomplete](./images/Autocomplete.jpg?raw=true 'Autocomplete')
```

Comment:
This will be broken on the npm page, that's why it was an absolute URL

Reply:
I think the links were broken because of the monorepo structure. Now that we've gotten rid of it, I think npm can handle relative links.

(It might not. I remember that there were npm vs monorepo issues, but I'm not 100% sure if this was one of them.)

Reply:
It might be.  Someone can followup on this but I don't think this is the time to do that investigation.  Right now it is broken completely so let us fix that first.